slack-plugin/version-templates/KotlinBuildConfig.kt

```diff
@@ -1,9 +0,0 @@
-@file:JvmName("KotlinBuildConfig")
```

Comment:
Why is this removed? Still seeing its reference here
https://github.com/slackhq/slack-gradle-plugin/pull/677/files#diff-c0dfa6bc7a8685217f70a860145fbdf416d449eaff052fa28352c5cec1a98c06R180.

Reply:
That's different than this file actually, they're just named the same. That other def is here: https://github.com/slackhq/slack-gradle-plugin/pull/677/files#diff-c0dfa6bc7a8685217f70a860145fbdf416d449eaff052fa28352c5cec1a98c06R122